elliott

```diff
@@ -348,11 +353,17 @@ advisory.
 @click.option("--id", type=int, metavar='BUGID',
               multiple=True, required=False,
               help="Bugzilla IDs to add, conflicts with --auto [MULTIPLE]")
+@click.option("--old",
+              required=False,
+              help="Older payload to compare against")
+@click.option("--new",
+              required=False,
+              help="Newer payload to compare agaist")
 @click.option("--flag", metavar='FLAG',
               required=False, multiple=True,
               help="Optional flag to apply to found bugs [MULTIPLE]")
 @pass_runtime
-def find_bugs(runtime, advisory, default_advisory_type, auto, status, id, flag):
+def find_bugs(runtime, advisory, default_advisory_type, mode, status, id, old, new, flag):
```

Comment:
Putting all those comments together, we'd have examples like this:

*Bugs between two payloads:* 
`elliott --group-openshift-4.1 find-bugs --mode diff --between 4.0.0-0.1 4.0.0-0.1` 

*List 4.1 bugs that would be attached (implicit `mode=list`):*
`elliott --group-openshift-4.1 find-bugs`

*Sweep bugs into 123456:*
`elliott --group-openshift-4.1 find-bugs --mode sweep --add 123456`

*Sweep bugs into default advisory:*
`elliott --group-openshift-4.1 find-bugs --mode sweep --use-default-advisory rpm`

Once you finalize this then some of those other examples will want to be looked at again.